### ISSUE TITLE
Warzone, Flashpoint ... IsReady property/event

### ISSUE THREAD

**Issue by exodus444** [BosslandGmbH/BuddyWing#30]
If such event could be exposed through the API I would write a plugin for desktop notification and pushbullet to mobile devices. I am sure a lot of players would love to have such a feature...

Edited: GameEvents seems like a good place for WarzoneIsReady, FlashpointIsReady etc events.

**Comment by aevitas** [comment]
I'm not sure I'm completely catching on your drift here - is it API that triggers whenever a warzone or flashpoint queue pops that you're looking for?

**Comment by exodus444** [comment]
That's right. Looking for event which can be used to determine that in game popup for entering the WZ or FP is up.
Would be even better if there was also a method to enter the warzone/flashpoint programmatically.

**Comment by aevitas** [comment]
All right, I'll see what I can find for this.

**Comment by pjmagee** [comment]
:+1:

**Comment by aevitas** [comment]
Alright, for Warzones, there is a function called `OnWarzoneConfirmationRequest`, but this function is an event in the game itself. With us being external, we can't exactly subscribe to events in the game, at least not in a way that I know of. For Flashpoints, such a function does not seem to exist.

This means that currently, this can't be implemented in the form I initially planned on doing - an event in `BotEvents` that users of the API could subscribe to and get notifications from. 

We may be able to find something in the GUI indicating that a flashpoint or warzone queue has popped, but that'll have to be polled and will require a significant amount of time to reverse the whole GUI component system introduced in a later expansion.

We'll be exploring possibilities for this, but I'll have to move this beyond the initial April milestone, because that simply didn't end up being realistic.

**Comment by exodus444** [comment]
Thanks for the efforts! I hope you will be able to find some other way!
